### DISCUSSION TITLE
2024/12/17/Where-are-those-algorithms/

### DISCUSSION THREAD

**Discussion by yuhiri-s-giscus** [talentestors/talentestors.github.io#56] [Announcements]
# 2024/12/17/Where-are-those-algorithms/

https://yuhiri.me/2024/12/17/Where-are-those-algorithms/

**Comment by talentestors** [comment]
Linux中国：
标准的 Markdown 格式的数据： https://github.com/Linux-CN/archive/releases/tag/release
Huggingface 数据集： https://huggingface.co/datasets/linux-cn/archive
